Specifications/QIR/Data-Types.md

```diff
@@ -99,6 +109,8 @@ simple types:
 | __quantum__rt__result_reference   | `void(%Result*)`         | Increments the reference count of a Result pointer. |
 | __quantum__rt__result_unreference | `void(%Result*)`         | Decrements the reference count of a Result pointer and releases the result if appropriate. |
 
+For `equal`, if a `%Result*` parameter is null, a runtime failure should occur.
```

Comment:
I'd suggest to use full function names all the time, as it makes the specifications more searchable.

Reply:
Done.